src/components/ShoppingLists/ShoppingLists.js

```diff
@@ -44,42 +67,99 @@ const ShoppingLists = () => {
     {
       key: "name",
       label: "Name",
-      renderItem: (row) =>
-        row?.nameAllLocales ? row?.nameAllLocales[0]?.value : "",
+      renderItem: (row) => {
+        if (updateVisibility.status && updateVisibility.rowId == row.id)
+          return (
+            <TextInput
+              value={updateVisibility.name}
+              onChange={(event) =>
+                setUpdateVisibility({
+                  ...updateVisibility,
+                  name: event.target.value,
+                })
+              }
+            />
+          );
+
+        return row?.nameAllLocales ? row?.nameAllLocales[0]?.value : "";
```

Comment:
You might want to look at the l10n transformers to help rendering a localized field.

https://github.com/commercetools/merchant-center-application-kit/tree/main/packages/l10n